src/components/layout/container.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
+import * as Styled from './styles'
```

Comment:
```suggestion
import * as S from './styles'
```

Reply:
and the rest files

Reply:
What's the reason for that `Styled` seems like standart way how to work with SC.

Reply:
It's specified in the CONTRIBUTE.md. There was a discussion about in the slack channel. Both versions look reasonable, the one from CONTRIBUTE.md is shorter. If you want, you can open the discussion in slack channel to change the "rule".